### PR TITLE
converted changelist.txt to markdown format

### PR DESCRIPTION
**Pull request by enricosada**
rebased #181 to master

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    Copyright (c) Microsoft Open Technologies, Inc.  All Rights Reserved.  
+
+    Licensed under the Apache License, Version 2.0.  
+    See License.txt in the project root for license information.
+
+
+# Visual F<span>#</span>
+All notable changes to this project will be documented in this file.
+
+This project **doesn't** use [Semantic Versioning](http://semver.org/)
```
Comment (latkin) [inline CHANGELOG.md]: These 2 lines are not needed and should be removed

**Comment by latkin** [comment]
Still a few things I don't like here:
- Pointers to codeplex bugs have been removed (please add back, ideally with link to codeplex issue)
- No need for header lines, as commented
- Adding loud tags (`FEATURE`, `PERFORMANCE`, `REMOVED`) is distracting and should be reverted
- No need to include F# 4.0 info yet, especially not in master branch

The file should just be translated as closely as possible to markdown, maybe adding a few links to bugs. No need to re-design the whole thing.

**Comment by latkin** [comment]
Whoops, didn't see that the codeplex links were just moved to the end of the lines. Have updated my comment.

**Comment by enricosada** [comment]
i'll update this soon, sry for delay

**Comment by enricosada** [comment]
done, [preview on my branch](https://github.com/enricosada/visualfsharp/blob/changelog_md/CHANGELOG.md)

**Comment by latkin** [comment]
Thanks, looks good to me.

**Pull request merged**